### ISSUE TITLE
Can not assign timeout > 240s

### ISSUE THREAD

**Issue by warrenwyf** [wg/wrk#127]
Got error message 'Segmentation fault: 11'  after setting timeout larger than 240s like this:
```bash
wrk -t 4 -c 200 -d 30s --timeout 300s http://localhost
```

**Comment by wg** [comment]
Hi @warrenwyf, is this on OS X? It looks like a conflict between the amount of RAM required for the statistics and LuaJIT's address space limitations. Until the new LuaJIT GC is released I'd suggest reducing your timeout. Do your requests really take up to 5 minutes to complete?

**Comment by warrenwyf** [comment]
Yes, test is on OSX. Just report the issue, however I do not need such a long timeout.

**Comment by wg** [comment]
Great, thank you! I'll leave this open for documentation purposes.

**Comment by avshalomman** [comment]
@warrenwyf I know it's been a while, but just came across this thread and wanted to share my solution.
(I hit the seg fault when trying to run a lua script per request).

build wrk from source, adding this line to the makefile:

 LDFLAGS += -pagezero_size 10000 -image_base 100000000

**Comment by timz1290** [comment]
Any sense of when the new LuaJIT GC will be released?

**Comment by timz1290** [comment]
@avshalomman - your solution worked
